src/instructlab/sdg/configs/skills/contexts.yaml

```diff
@@ -19,5 +19,3 @@ generation: |
 
 start_tags: ["[Start of Context]"]
 end_tags: ["[End of Context]"]
-
-sys_prompt: You are a very knowledgeable AI Assistant that will faithfully assist the user with their task.
```

Comment:
I think this is unrelated? If so, I'd prefer to deal with it separately. See #184